### DISCUSSION TITLE
Validate a cron pattern without actually scheduling it

### DISCUSSION THREAD

**Discussion by mactyr** [Hexagon/croner#222] [Ideas]
As someone mentioned in [this comment](https://github.com/Hexagon/croner/discussions/166#discussioncomment-5981290), it would be helpful to have a clean way to validate a cron pattern without actually scheduling it. I think I can get close enough currently by doing:
```
try {
  Cron(pattern, {stopAt: new Date()});
} catch (err) {
  // whatever I want to do if the pattern is invalid
}
```
But this is kind of messy and verbose relative to something like `Cron.isValid(pattern)` that would return a `boolean`.

**Comment by Hexagon** [comment]
Great idea 👍
